### PR TITLE
Laravel 11 support

### PR DESCRIPTION
**Pull request by dcodeaaron**
Existing v1.x releases to be retained in a renamed `main` branch to `1.x`. All `1.x.x` tags to come from this branch.

New project primary branch `2.x` will this branch merged into it providing the basis for the future and Laravel v11+ support. All `2.x.x` tags to come from this branch.

For more information see starter project Laravel 11 PR for details: https://github.com/DCODE-GROUP/project-starter/pull/50

### CODE REVIEW

**Comment by lionslair** [comment]
Same suggestion as others. Rename the old branch to 10.x and new 11.x or similar so if changes are needed to the old later we still can.

File: build/phpstan/cache/PHPStan/4c/be/4cbec5eaf9a9ea3bde1fe4e7bcc8c997529f1bc0.php [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<?php declare(strict_types = 1);
```
Comment (andrewsirianni) [inline build/phpstan/cache/PHPStan/4c/be/4cbec5eaf9a9ea3bde1fe4e7bcc8c997529f1bc0.php]: Should this be here?

**Review by andrewsirianni — COMMENTED** [review]
Has the build directory accidentally been included?

File: build/phpstan/cache/PHPStan/4c/be/4cbec5eaf9a9ea3bde1fe4e7bcc8c997529f1bc0.php (hunk 1 above)
Comment (dcodeaaron) [reply build/phpstan/cache/PHPStan/4c/be/4cbec5eaf9a9ea3bde1fe4e7bcc8c997529f1bc0.php]: removed

**Pull request merged**